scripts/install

```diff
@@ -57,6 +57,10 @@ ynh_script_progression --message="Validating installation parameters..." --weigh
 final_path=/opt/yunohost/$app
 test ! -e "$final_path" || ynh_die --message="This path already contains a folder"
 
+if [ $encryption -eq 1 ]; then
+  encryption="true"
+fi
```

Comment:
This makes me a little cringe as in "why not set it to false when the value is different than 1 (ie 0) ? :D

Reply:
Yes, you are right here, this part I did it a while ago and kinda missed to revisit it and finish it properly 😮‍💨